### PR TITLE
fix(utils): 14472 createAsyncAtom - incorrect processing error in fetcher

### PR DESCRIPTION
**Pull request by Akiyamka**
Regular throw error in fetcher (not canlcel) of AsyncAtom not reset abortController, and next request was ignored

### CODE REVIEW

File: src/utils/atoms/createAsyncAtom/createAsyncAtom.ts [hunk 1]
```diff
@@ -165,6 +169,8 @@ export function createAsyncAtom<
               log('5.1.B.1. Error');
               dispatch(create('_error', params, ABORT_ERROR_MESSAGE));
             } else if (ctx.abortController === abortController) {
+              ctx.abortController = null;
```
Comment (dqunbp) [inline src/utils/atoms/createAsyncAtom/createAsyncAtom.ts]: `delete ctx.abortController;` will do the same as `ctx.abortController = null;`?

Comment (Akiyamka) [reply src/utils/atoms/createAsyncAtom/createAsyncAtom.ts]:
```suggestion
              delete ctx.abortController;
```

Comment (Akiyamka) [reply src/utils/atoms/createAsyncAtom/createAsyncAtom.ts]: yep

Comment (propakov) [reply src/utils/atoms/createAsyncAtom/createAsyncAtom.ts]: let's do it in one approach. 
If we go with delete, could you also please update in line 201?

**Review by propakov — APPROVED** [review]

**Pull request merged**